kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -871,49 +871,73 @@ object KyuubiConf {
       .intConf
       .createWithDefault(100)
 
-  val SERVER_STATE_STORE_CLASS: ConfigEntry[String] =
-    buildConf("kyuubi.server.state.store.class")
-      .doc("Fully qualified class name for server state store.")
+  val METADATA_STORE_CLASS: ConfigEntry[String] =
+    buildConf("kyuubi.metadata.store.class")
+      .doc("Fully qualified class name for server metadata store.")
       .version("1.6.0")
       .stringConf
-      .createWithDefault("org.apache.kyuubi.server.statestore.jdbc.JDBCStateStore")
+      .createWithDefault("org.apache.kyuubi.server.metadata.jdbc.JDBCMetadataStore")
 
-  val SERVER_STATE_STORE_CLEANER_ENABLED: ConfigEntry[Boolean] =
-    buildConf("kyuubi.server.state.store.cleaner.enabled")
-      .doc("Whether to clean the state store periodically. If it is enabled, Kyuubi will clean" +
-        " the state information that is in terminate state with max age limitation.")
+  val METADATA_CLEANER_ENABLED: ConfigEntry[Boolean] =
+    buildConf("kyuubi.metadata.cleaner.enabled")
+      .doc("Whether to clean the metadata periodically. If it is enabled, Kyuubi will clean the" +
+        " metadata that is in terminate state with max age limitation.")
       .version("1.6.0")
       .booleanConf
       .createWithDefault(true)
 
-  val SERVER_STATE_STORE_MAX_AGE: ConfigEntry[Long] =
-    buildConf("kyuubi.server.state.store.max.age")
-      .doc("The maximum age of state info in state store.")
+  val METADATA_MAX_AGE: ConfigEntry[Long] =
+    buildConf("kyuubi.metadata.max.age")
+      .doc("The maximum age of metadata.")
       .version("1.6.0")
       .timeConf
       .createWithDefault(Duration.ofDays(3).toMillis)
 
-  val SERVER_STATE_STORE_CLEANER_INTERVAL: ConfigEntry[Long] =
-    buildConf("kyuubi.server.state.store.cleaner.interval")
-      .doc("The interval to clean state store.")
+  val METADATA_CLEANER_INTERVAL: ConfigEntry[Long] =
+    buildConf("kyuubi.metadata.cleaner.interval")
+      .doc("The interval to clean metadata.")
       .version("1.6.0")
       .timeConf
       .createWithDefault(Duration.ofMinutes(30).toMillis)
 
-  val SERVER_STATE_STORE_SESSIONS_RECOVERY_PER_BATCH: ConfigEntry[Int] =
-    buildConf("kyuubi.server.state.store.sessions.recovery.per.batch")
-      .doc("The number of sessions to recover from state store per batch.")
+  val METADATA_SESSIONS_RECOVERY_PER_BATCH: ConfigEntry[Int] =
+    buildConf("kyuubi.metadata.sessions.recovery.per.batch")
```

Comment:
I do not quite undertand the meaning of this conf

Reply:
how about remove it? It should be fine to get all the batch metadata for recovery.

Reply:
+1